generated/niscope/unit_tests/mock_helper.py

```diff
@@ -273,17 +273,11 @@ def niScope_Fetch(self, vi, channel_list, timeout, num_samples, wfm, wfm_info):
         if self._defaults['Fetch']['Wfm'] is None:
             raise MockFunctionCallError("niScope_Fetch", param='Wfm')
         a = self._defaults['Fetch']['Wfm']
-        import sys
```

Comment:
we should just import this at the top of mock_helper.py.

Reply:
Done